pkg/asset/ignition/machine/master.go

```diff
@@ -63,3 +63,12 @@ func (a *Master) Name() string {
 func (a *Master) Files() []*asset.File {
 	return a.files
 }
+
+// Load returns the master ignitions from disk.
+func (a *Master) Load(p asset.FilePatternFetcher) (found bool, err error) {
+	files, found, err := p("master-[0-9]*.ign")
```

Comment:
This pattern is not matching the right filenames for master ignitions. It accepts filenames of the form "master-" followed by a digit followed by anything and ending with the extension "*.ign". For example, it will match the filename `master-0bad.ign`.